src/components/FeedbackError/FeedbackError.tsx

```diff
@@ -12,23 +12,58 @@ import { animationVariants } from './FeedbackError.data';
 
 function FeedbackError(): ReactNode {
   const errorMessage = useError((state) => state.errorMessage);
+  const [showErrors, setShowErrors] = useState(false);
+
+  const errors = [''];
+
+  const renderErrorDropdown = (): ReactNode => (
+    <AnimatePresence>
+      <motion.ul
+        animate="visible"
+        className={scss.wrapperErrorList}
+        exit="hidden"
+        initial="hidden"
+        variants={animationVariants}
+      >
+        {errors.map((error) => (
+          <li key={error}>{error}</li>
+        ))}
+      </motion.ul>
+    </AnimatePresence>
+  );
 
   const renderError = (): ReactNode => (
     <motion.div
       animate="visible"
-      className={scss.wrapper}
       data-testid="error-container"
       exit="hidden"
       initial="hidden"
       variants={animationVariants}
     >
-      <Icon className={scss.errorIcon} icon="alert" size={12} />
-      <p className={scss.errorMessage}>{errorMessage}</p>
+      <div className={scss.errorSection}>
+        <div className={scss.errorContainer}>
+          <Icon className={scss.alertIcon} icon="alert" size={20} />
+          <div className={scss.errorMessageContainer}>
+            <p className={scss.errorMessage}>{errorMessage}</p>
+          </div>
+          <button
+            className={scss.dropDownIconContainer}
+            onClick={() => setShowErrors((prevState) => !prevState)}
+            type="button"
+          >
+            <Icon
+              className={scss.dropDownIcon}
+              icon="DropDownArrow"
+              size={10}
+            />
+          </button>
+        </div>
+        {showErrors && renderErrorDropdown()}
+      </div>
     </motion.div>
   );
-  return (
-    <AnimatePresence>{errorMessage ? renderError() : null}</AnimatePresence>
-  );
+
+  return errorMessage ? renderError() : null;
```

Comment:
```suggestion
  return errorMessage && renderError();
```
o problema pro ale ai nao era o `if ternário`, era o lugar do `AnimatedPresence`